### PR TITLE
Reorganize to hide internals and add start.sh

### PR DESCRIPTION
**Pull request by synth-mania**
I put all the project files into a src/ folder to hide the internals and improve the user experience a bit while making things looks a bit neater.

This involved

- creating a shell script that automatically creates, activates, and deactivates the venv. It also makes sure requirements are up to date
- moved all the .py files into the new src/ folder
- made all the imports between .py files relative so it would work as a package
- renamed Web_LLM.py `__main__.py` and created `__init__.py` so src is treated as a package
- update `README.md` to specify that modelfile can be created in the project root

I also added a small note in the 'configuration' section of the README to dispel any confusion on which model name should be copied into llm_config.py

The program works exactly as it did before, just a little easier to use now.

Let me know if you have any questions.

### CODE REVIEW

**Comment by kjenney** [comment]
Thanks @synth-mania. I noticed the following issues with your code:
1. It doesn't account for the model not being defined in llm_config. I get a 404 error from Ollama if it's not defined.
2. `start.sh` needs to have executable permissions or it fails to run with `./start.sh`.
3. `start.sh` is formatted in CLRF which doesn't work well in Linux and Mac systems. I had to convert it using dos2unix on my Mac.
4. bash is not always at `/bin`. Use env to locate where bash is on a users system.

File: start.sh [hunk 1]
```diff
@@ -0,0 +1,13 @@
+#!/bin/bash
+
+if [ ! -d "venv" ]; then
+    python3 -m venv venv
+fi
+
+source venv/bin/activate
```
Comment (kjenney) [inline start.sh:7]: This should be moved to line 4

File: start.sh [hunk 2]
```diff
@@ -0,0 +1,13 @@
+#!/bin/bash
```
Comment (kjenney) [inline start.sh:1]: This should use env to locate bash

File: start.sh (hunk 1 above)
Comment (kjenney) [reply start.sh:7]: Maybe try adding "ollama create" here?

**Review by kjenney — CHANGES_REQUESTED** [review]

**Comment by synth-mania** [comment]
@kjenney It's interesting that you had to reformat start.sh, especially because I wrote that on my fedora workstation. I'm not going to be available for the next couple hours - traveling for Thanksgiving, but I'll try to fix these issues as soon as possible.

**Comment by TheBlewish** [comment]
Thanks for the work people! Please let me know when you have it functional, I don't wanna repeat deleting the entire functionality of the main by a merge that wasn't great again haha
I'll check this out in a day or so hopefully it'll be working better by then!

**Pull request closed without merging**